ansible/roles/demo_prep/tasks/main.yml

```diff
@@ -50,3 +50,19 @@
 
   - name: delete imagestreams
     shell: "{{ oc_cmd }} delete -n {{ demo_project_name }} is demoprep python-35-centos7"
+
+  - name: check if the rds postgres demo instance exists
+    shell: "/usr/local/bin/kubectl --kubeconfig=/root/.kube/service-catalog.config get instance -n {{ demo_project_name }} --no-headers=true | awk '{ print $1}' | grep -E '^rds-postgres( |$)' | cat"
```

Comment:
Let's use {{ kubectl_cmd }} instead of /usr/local/bin/kubectl
We will be installing kubectl to a few different possibilities, so helps if we don't hardcode the path.